Marlin/ubl_G29.cpp

```diff
@@ -996,16 +1018,27 @@
   }
 
   bool g29_parameter_parsing() {
-    #if ENABLED(ULTRA_LCD)
-      LCD_MESSAGEPGM("Doing G29 UBL!");
-      lcd_quick_feedback();
-    #endif
+
+    LCD_MESSAGEPGM("Doing G29 UBL!");
+    lcd_quick_feedback();
+
+//  x_flag = code_seen('X') && code_has_value();
+//  y_flag = code_seen('Y') && code_has_value();
+//  x_pos = x_flag ? code_value_float() : current_position[X_AXIS];
+//  y_pos = y_flag ? code_value_float() : current_position[Y_AXIS];
+
+// Seriously???   You really think that you made the code better by changing that???
+// How about we try it like this instead:
```

Comment:
Please maintain the cascade, and you can text-message me if you have questions about any changes.